### PR TITLE
[local_auth] docs update

### PR DESCRIPTION
**Pull request by snwagbata**
The message that shows up is "This app was designed to support Touch ID. It has not been updated for Face ID." not "touch id"

## Description

The wrong error message is mentioned in the docs. This fixes that.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/local_auth/README.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ you need to also add:
 ```
 
 to your Info.plist file. Failure to do so results in a dialog that tells the user your
-app has not been updated to use TouchID.
+app has not been updated to use Face ID.
```
Comment (stuartmorgan) [inline packages/local_auth/README.md:135]: Could you update line 126 to use the correct spacing for Touch ID and Face ID so the documentation is consistent?

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
Thanks for the contribution! This will also need a version update in pubspec.yaml and CHANGELOG.md so that we can publish it, since the published version of the README is what most people see.

We apologize for the long delay in triaging this PR. We’re in the process of overhauling our PR triage system to respond much more quickly, as well as working through the backlog.

**Comment by snwagbata** [comment]
Hi @stuartmorgan, since the version of local_auth has changed since this PR, do I copy those changes into my branch and commit it

**Comment by stuartmorgan** [comment]
You would merge master into your branch.

**Comment by snwagbata** [comment]
@stuartmorgan I've merged master into my branch and updated the changelog

**Comment by cyanglaz** [comment]
@snwagbata Looks like there's still some conflicts between master and this PR :)

**Comment by snwagbata** [comment]
@cyanglaz it seems someone made an update to the CHANGELOG sometime in the past few days, I just resolved that conflict.

**Review by cyanglaz — DISMISSED** [review]
Since 1.1.4 has already been released, we would require a new release for this. so 1.1.5

**Comment by snwagbata** [comment]
@cyanglaz hi, I just updated the CHNAGELOG and pubspec for the new release

**Review by stuartmorgan — APPROVED** [review]
LGTM!

**Pull request merged**